level00/PropertyTests.hs

```diff
@@ -112,7 +113,44 @@ genTree genKV = fromList <$> Gen.list (Range.linear 0 100) genKV
 genMyBTreeVal :: MonadGen m => m (Int, Char)
 genMyBTreeVal = liftA2 (,) (Gen.int (Range.linear (-100) 100)) (Gen.enum 'a' 'z')
 
--- We're not ready to write a property test for inserting values into our binary
+-- We can also use property based testing to ensure that when we
+-- implement a typeclass instance, that implementation will comply
+-- with the laws of that typeclass. This is important as often the
+-- laws of a typeclass cannot be expressed in the types.
+--
+-- Implement the 'Eq' instance for 'MyBTree' and then complete the
```

Comment:
The Eq instance is not unique- should we talk about trees being equal if they have the same keys/vals instead of identical structure?

Reply:
Not sure how much detail to go into ? I've updated the instructions a bit...

Also does my implementation of the property look sufficient ? Bumped it to a hundred thousand tests (lulz) and it was still okay so I'm somewhat confident.